examples/external_speed_profile.py

```diff
@@ -16,45 +17,58 @@
 const_switch_gen = rg.SwitchedReferenceGenerator(const_sub_gen,
                                                  super_episode_length=(1000, 2000))
 
-# speed profiles
+# possible speed profiles
 sinus_lambda = (lambda t,f,amp, b: amp * np.sin(2*np.pi*f*t) + b)
 constant_lambda = (lambda value: value)
 triangle_lambda = (lambda t, amp, f, b: amp * signal.sawtooth(2 * np.pi * f * t,
-                                                               width=0.5) + b)
+                                                             width=0.5) + b)
 saw_lambda = (lambda t, amp, f, b: amp * signal.sawtooth(2 * np.pi * f * t) + b)
 
+# usage of a random load initializer is only recommended for the
+# ConstantSpeedLoad, due to the already given profile by an ExternalSpeedLoad
+load_init = {'random_init': 'uniform'},
+
 # external speed profiles can be given by an ExternalSpeedLoad, ConstantSpeedLoad
-# initial values are defined in the class
-ext_load = ExternalSpeedLoad(omega_initial=5, speed_profile=sinus_lambda,
-                             amp=30, f=2, b=30)
+# the initial value of the external load should be match with the profile
+# offset to avoid complicated spikes after a new episode start
```

Comment:
The hints given in these comments do not match the intended behavior of the externalSpeedLoad. Information on that are noted my other annotations.